### PR TITLE
add seissolxdmf module

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**

### CODE REVIEW

File: seissolxmdf/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+fn = 'test-fault.xdmf'
+# Number of cells
+nElements = sx.ReadNElements(fn)
+# Read time step
+dt = sx.ReadTimeStep(fn)
+# Read number of time steps
+ndt = sx.ReadNElements(fn)
+# load geometry array as a numpy array of shape ((nodes, 3))
+geom = sx.ReadGeometry(fn)
+# load connect array as a numpy array of shape ((nElements, 3 or 4))
```
Comment (sebwolf-de) [inline seissolxmdf/README.md]: Can you explain what the entries of the connect array are?

**Review by sebwolf-de — COMMENTED** [review]
Is seissolxdmf/seissolxdmf/seissolxdmf.py is just a copy of seissol xdmf writer from your personal repo, right?

**Comment by Thomas-Ulrich** [comment]
> Is seissolxdmf/seissolxdmf/seissolxdmf.py is just a copy of seissol xdmf writer from your personal repo, right?

Hi,
Kind of, it is based on the master branch, but I refactored it.
My personal repo will be deactivated once this is merged.

Note that it is meant to be a pip module.

File: seissolxmdf/README.md (hunk 1 above)
Comment (Thomas-Ulrich) [reply seissolxmdf/README.md]: ok

**Comment by sebwolf-de** [comment]
Okay, I have reviewed seissolxmdf.py superficially and found no problems. As it has been in use before I see no problem in merging. Anyways, I have no experience with pip modules other than using them. Maybe ask @ravil-mobile for a review in that direction.

**Pull request merged**